cocos/animation/marionette/state-machine/state-machine-eval.ts

```diff
@@ -626,95 +589,64 @@ class TopLevelStateMachineEvaluation {
     }
 
     /**
-     * Searches for a transition which should be performed
-     * if current node update for no more than `deltaTime`.
-     * @param deltaTime
+     * Searches for a transition which should be performed.
+     * @param sourceState The transition source state.
+     * @param useFromPort See `this._matchTransition`.
```

Comment:
There is only one parameter.

Reply:
Solved https://github.com/cocos/cocos-engine/pull/14990/commits/1014b217fea545b78a27b644d43bfa3aab642536